webauthn/helpers/verify_safetynet_timestamp.py

```diff
@@ -0,0 +1,21 @@
+import time
+
+
+def verify_safetynet_timestamp(timestamp_ms: int) -> None:
+    """Handle time drift between an RP and the Google SafetyNet API servers with a window of
+    time within which the response is valid
+    """
+    # Buffer period in ms
+    grace_ms = 10 * 1000
```

Comment:
For a lower level package/library - I would suggest making constants like this configurable